packages/cli/src/tools/deleteFile.ts

```diff
@@ -0,0 +1,4 @@
+import {unlink} from 'fs';
+import {promisify} from 'util';
+
+export const deleteFile = promisify(unlink);
```

Comment:
We can use `fs.unlinkSync` and be done, without creating new files, why making it async?

Reply:
It's a lot easier to mock if it's in a separate file. Also I'll be using this again for the android dependencies.

Reply:
Fair call. Still doesn't need to be async, right? :)

Reply:
Doesn't have to, but I think it's better if it is.
This is a blocking call and we don't know how long it will take. We have a spinner in the UI and if it stops for whatever reason it will provide a bad experience to the user.

Another reason (and this is a potential) is if fixes start running in parallel to speed up the process this will be blocking everything.

The difference for any "consumer" of this method is just adding `await` in front of it which I don't think it's a big deal. IMO the pros outweigh the cons, especially with not blocking the UI 😊